docs/docs-content/palette-cli/commands/fips-validate.md

```diff
@@ -0,0 +1,341 @@
+| ------------------------------ | ----------------------------- |
+| `TLSv1.3`                      | The TLS version.              |
+| `x1303`                        | The cipher suite HEX code.    |
+| `TLS_CHACHA20_POLY1305_SHA256` | The name of the cipher suite. |
+| `ECDH 253`                     | The key exchange algorithm.   |
+| `ChaCha20`                     | The encryption algorithm.     |
+| `256`                          | The encryption key length.    |
+| `TLS_CHACHA20_POLY1305_SHA256` | The authentication algorithm. |
+
+The service endpoint is using the `TLSv1.3` version with `TLS_CHACHA20_POLY1305_SHA256` cipher suit. The TLS version and
+the cipher suite is not considered FIPS compliant and the status is marked as `FAILED`.
+
+A successful check will not have a finding identifier. The following is an example of a successful check.
+
+```text
+---------------------------------------------------------------
+endpoint: palette-webhook-service.palette-system
+note: all validations passed
+---------------------------------------------------------------
+```
+
+The end of the report contains a summary of the scan results. The summary includes all the service endpoints scanned and
+the status of each endpoint. Below is an example of the summary section.
+
+```text
+┌────────────────────────────────────────────────────────────────┬────────┐
+│ ENDPOINT                                                       │ STATUS │
+├────────────────────────────────────────────────────────────────┼────────┤
+│ capa-webhook-service.capi-webhook-system                       │ passed │
+│ capi-kubeadm-bootstrap-webhook-service.capi-webhook-system     │ passed │
+│ capi-kubeadm-control-plane-webhook-service.capi-webhook-system │ passed │
+│ capi-webhook-service.capi-webhook-system                       │ passed │
+│ capvc-webhook-service.capi-webhook-system                      │ failed │
+│ cert-manager-webhook.cert-manager                              │ passed │
+│ metrics-server.cluster-65e4cb59cbfc84ea5877af4c                │ passed │
+│ oldrelease-vc1.cluster-65ea4b3b9fe382461d51fbb3                │ failed │
+│ oldrelease-vc1-headless.cluster-65ea4b3b9fe382461d51fbb3       │ failed │
+│ oldrelease-vc1-lb.cluster-65ea4b3b9fe382461d51fbb3             │ failed │
+│ kubernetes.default                                             │ passed │
+│ palette-webhook-service.palette-system                         │ passed │
+│ 10-0-3-97.kubernetes.default:2380                              │ failed │
+│ 10-0-3-97.kubernetes.default:10250                             │ passed │
+└────────────────────────────────────────────────────────────────┴────────┘
+```
+
+## Clean
+
+The `clean` subcommand removes the FIPS validation resources from your clusters. The command is intended to be used in
+scenarios where the `images` or `services` subcommands fail to clean up the resources from the cluster. The `clean`
```

Comment:
Why would they fail? Perhaps they are still in use or the cluster is no longer healthy?